services/net/image/ImageAction.cs

```diff
@@ -73,7 +73,7 @@ public ImageAction(IKafkaMessenger producer, IApiService api, IOptions<ImageOpti
         this.Logger.LogDebug("Performing ingestion service action for data source '{name}'", manager.Ingest.Name);
 
         // Extract the ingest configuration settings.
-        var inputFileCode = String.IsNullOrWhiteSpace(manager.Ingest.GetConfigurationValue("code")) ? manager.Ingest.Source?.Code : manager.Ingest.GetConfigurationValue("code");
+        var inputFileCode = String.IsNullOrWhiteSpace(manager.Ingest.GetConfigurationValue("code")) ? manager.Ingest.Source?.Code : manager.Ingest.SourceConnection?.GetConfigurationValue("code");
```

Comment:
Am I misunderstanding the `code` property?  I would think this is should be `manager.Ingest.GetConfigurationValue("code")`.

Reply:
I reverted the line to what you suggested here just before my last push. Not sure why it didn't apply the change. I'll check.